src/ansys/hps/client/auth/api/auth_api.py

```diff
@@ -21,20 +21,16 @@
 # SOFTWARE.
 """Module providing the Python interface to the Authorization Service API."""
 
-from typing import List
+from typing import Dict, List
 
-from keycloak import KeycloakAdmin
+from ansys.hps.client.client import Client
 
 from ..resource import User
 from ..schema.user import UserSchema
 
 
 class AuthApi:
-    """Provides the Python interface to the Authorization Service API.
-
-    Admin users with the Keycloak "manage-users" role can create
-    users as well as modify or delete existing users. Non-admin users are only allowed
-    to query the list of existing users.
+    """Minimal wrapper around the Keycloak API to query users info.
```

Comment:
```suggestion
    """Provides a minimal wrapper around the Keycloak API to query user information.
```
All class descriptions should start with a verb ending in "s" or "es" for consistency (and per our standard).